release/create_candidate_pr.py

```diff
@@ -0,0 +1,86 @@
+# pylint: disable=redefined-outer-name, invalid-name
+
+"""
+Creates a new release candidate branch and makes a pull request
+for it into the 'release' branch.
+"""
+
+from __future__ import print_function
+
+import argparse
+
+from release.github_api import GithubApi
+from release.utils import default_expected_release_date
+
+
+class NoValidCommitsError(Exception):
+    """
+    Error indicating that there are now commits that do not have status
+    problems.
+    """
+    pass
+
+
+def _build_parser():
+    """ Creates an option parser for this command """
+    expected_date = default_expected_release_date()
+    result = argparse.ArgumentParser(description="""
```

Comment:
This can be made a bit easier to read by using spacing around arguments. See https://github.com/edx/repo-tools/blob/master/monthly_pr_stats.py#L87